tsfile/src/main/java/org/apache/iotdb/tsfile/encoding/decoder/IntRleDecoder.java

```diff
@@ -116,18 +58,14 @@ protected void readNumberInRle() throws IOException {
 
   @Override
   protected void readBitPackingBuffer(int bitPackedGroupCount, int lastBitPackedNum) {
-    currentBuffer = new int[bitPackedGroupCount * config.RLE_MIN_REPEATED_NUM];
+    currentBuffer = new Integer[bitPackedGroupCount * TSFileConfig.RLE_MIN_REPEATED_NUM];

```

Comment:
I think using Integer is slow.

Reply:
These kinds of modifications will be revoked.